functions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+function siteURL()
+{
+    $protocol = (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' || $_SERVER['SERVER_PORT'] == 443) ? "https://" : "http://";
+    $domainName = $_SERVER['HTTP_HOST'] . '/';
+    return $protocol . $domainName;
```

Comment:
I think that this function should have an optional input in its function for a folder string, as this function currently does not work on devlb. On devlb this function looks for devlb.nationalarchives.gov.uk/header.php, whereas if you could specify folders for it to look for then I can point it to the right folders (devlb.nationalarchives.gov.uk/phpapps/apprentice-folder/ash/latin/header.php).

However, if you are using localhost this function currently works fine.